auth/src/api/services/user-service.ts

```diff
@@ -63,17 +64,17 @@ export class UserService implements UserHandler {
         const userObj = await this.getUserCollectionFromDatastore('username', username);
         Logger.debug('User object from datastore: ', userObj);
         if (Object.keys(userObj).length > 1) {
-            throw new Error('Multiple users with same credentials!');
+            throw new AuthenticationException('Multiple users with same credentials!');
```

Comment:
this should not be an auth error (at least internally): it might be reported so, but this is a catastrophic failure since the db is inconsistent - can you log this seperately?